setup.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+''' Setup script, only usable via pip. '''
+
+import os
+import sys
+import glob
+
+from setuptools import setup, find_packages
+
+# ------------------------------------------------------------------------------
+# check python version. we need >= 2.7, <3.x
+if  sys.hexversion < 0x02070000 or sys.hexversion >= 0x03000000:
+    raise RuntimeError('%s requires Python 2.x (2.7 or higher)' % 'facts')
+
+
+# ------------------------------------------------------------------------------
+#
+# This copies the contents like examples/ dir under sys.prefix/share/$name
+# It needs the MANIFEST.in entries to work.
+base = 'share/%s' % 'facts'
+df = [('%s/'                      		% base, ['FACTS.py']),
+     ('%s/experiments/temp_exp'                 % base, glob.glob('experiments/temp_exp/*.yml')),
+     ('%s/experiments/temp_exp/input'           % base, glob.glob('experiments/temp_exp/input/*')),
```

Comment:
Would it be possible/worth separating the FACTS python module from the FACTS experiment repository?

Reply:
done!